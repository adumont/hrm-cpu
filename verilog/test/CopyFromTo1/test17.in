09 1d
26 c1
46 23
02 ad
28 00
21 f6
01 c2
46 94
48 b1
12 24
32 cf
18 8d
59 61
23 4d
33 40
36 b2
56 4e
59 50
42 76
55 4b
12 b1
20 bf
58 41
45 24
16 ee
31 c5
02 59
06 0c
18 ad
34 cd
32 f3
46 1d
31 4e
14 2b
49 ba
18 2e
09 8a
37 c3
26 e6
01 0e
56 80
58 a7
32 a4
07 ef
56 d4
09 22
54 47
