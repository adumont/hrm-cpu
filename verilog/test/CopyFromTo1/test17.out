1d
c1
23
ad
00
f6
c2
94
b1
24
cf
8d
61
4d
40
b2
4e
50
76
4b
b1
bf
41
24
ee
c5
59
0c
ad
cd
f3
1d
4e
2b
ba
2e
8a
c3
e6
0e
